src/averageAges.js

```diff
@@ -36,8 +40,18 @@ function calculateMenAverageAge(people, century) {
  *
  * @return {number}
  */
+
 function calculateWomenAverageAge(people, withChildren) {
-  // write code here
+  const mothersNames = [
+    ...new Set(people.map((person) => person.mother || 'not specified')),
+  ];
+  const arrayOfWoman = people.filter((person) => {
```

Comment:
It is better to not specify the type of variable in it's name, so it can be just `women`